tests/network/persistentips.go

```diff
@@ -0,0 +1,158 @@
+				},
+			}
+			By("Create NetworkAttachmentDefinition")
+			_, err := kubevirt.Client().NetworkClient().K8sCniCncfIoV1().NetworkAttachmentDefinitions(testsuite.GetTestNamespace(nil)).Create(context.TODO(), nad, metav1.CreateOptions{})
+			Expect(err).ShouldNot(HaveOccurred())
+
+			vm = libvmi.NewVirtualMachine(libvmifact.NewAlpine(
+				libvmi.WithNamespace(testsuite.GetTestNamespace(nil)),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithBridgeBinding("multus")),
+				libvmi.WithNetwork(libvmi.MultusNetwork("multus", "persistentips")),
+			), libvmi.WithRunning())
+
+			By("Creating VM using the nad")
+			vm, err = kubevirt.Client().VirtualMachine(vm.Namespace).Create(context.Background(), vm, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Wait for VMI to be created")
+			Eventually(refreshVMI).
+				WithPolling(time.Second).
+				WithTimeout(2 * time.Minute).
+				Should(Succeed())
+
+			By("Wait for VMI to be ready")
+			vmi = libwait.WaitUntilVMIReady(vmi, console.LoginToAlpine)
+
+			By("Check IPAMClaim get created")
+			ipamClaims, err := kubevirt.Client().IPAMClaimsClient().K8sV1alpha1().IPAMClaims(vm.Namespace).List(context.Background(), metav1.ListOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(ipamClaims.Items).ToNot(BeEmpty())
+
+		})
+
+		JustAfterEach(func() {
+			By("Check no ipamclaims leftovers after VM delete")
+			Expect(kubevirt.Client().VirtualMachine(vm.Namespace).Delete(context.Background(), vm.Name, metav1.DeleteOptions{})).To(Succeed())
+			Eventually(currentIPAMClaims).
+				WithTimeout(time.Minute).
+				WithPolling(time.Second).
+				Should(BeEmpty())
+		})
+
+		It("should keep ips after live migration", func() {
+			Expect(refreshVMI()).To(Succeed())
+			vmiIPsBeforeMigration := vmi.Status.Interfaces[0].IPs
+
+			migration := libmigration.New(vmi.Name, vmi.Namespace)
+			libmigration.RunMigrationAndExpectToCompleteWithDefaultTimeout(kubevirt.Client(), migration)
+
+			Expect(refreshVMI()).To(Succeed())
+			vmiIPsAfterMigration := vmi.Status.Interfaces[0].IPs
+			Expect(vmiIPsAfterMigration).To(ContainElements(vmiIPsBeforeMigration))
+		})
+
+		It("should keep ips after restart", func() {
+			Expect(refreshVMI()).To(Succeed())
+			vmiIPsBeforeRestart := vmi.Status.Interfaces[0].IPs
+			vmiUUIDBeforeRestart := vmi.UID
+
+			By("Re-starting the VM")
+			Expect(kubevirt.Client().VirtualMachine(vm.Namespace).Restart(
+				context.Background(),
+				vm.Name,
+				&v1.RestartOptions{},
+			)).To(Succeed())
+
+			By("Wait for a new VMI to be created")
+			Eventually(currentVMIUD).
+				WithPolling(time.Second).
+				WithTimeout(90 * time.Second).
+				ShouldNot(Equal(vmiUUIDBeforeRestart))
+
+			By("Wait for VMI to be ready after restart")
+			vmi = libwait.WaitUntilVMIReady(vmi, console.LoginToAlpine)
+
+			Expect(refreshVMI()).To(Succeed())
+			vmiIPsAfterRestart := vmi.Status.Interfaces[0].IPs
+			Expect(vmiIPsAfterRestart).To(ContainElements(vmiIPsBeforeRestart))
```

Comment:
shouldn't you use `ConsistOf` instead ? It's more accurate (in this particular situation I don't think it would matter though ...)

```golang
It("consist of vs contain elements", func() {
		Expect([]int{1, 2, 3, 4, 5}).To(ContainElements(1, 2, 3)) // this is true ...

		Expect([]int{1, 2, 3, 4, 5}).To(ConsistOf(1, 2, 3)) // this is false ...
	})
```